src/pages/Admin/Room/Edit/RoomEditModal.tsx

```diff
@@ -67,8 +67,12 @@ export const RoomEditModal: React.FC<RoomEditModalProps> = ({
   );
 
   const onSubmit = useCallback(() => {
-    return submitHandler(values, room.roomIndex!);
-  }, [room.roomIndex, submitHandler, values]);
+    const roomValues = {
```

Comment:
Just to make it more explicit/obvious:

```suggestion
    const roomValues: RoomData_v2 = {
```